src/TorchSharp/TorchVision/Normalize.cs

```diff
@@ -21,7 +21,8 @@ internal Normalize(double[] means, double[] stdevs, ScalarType dtype = ScalarTyp
                 this.means = this.means.to_type(dtype);
                 this.stdevs = this.stdevs.to_type(dtype);
             }
-            if (device != null) {
+
+            if (device != null && device.type != DeviceType.CPU) {
```

Comment:
@NiklasGustafsson Is this change fine? Here null check is not being removed.

Reply:
Yes, here is it is a (slight) performance benefit.